paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/ach/USBankAccountEmitters.kt

```diff
@@ -68,9 +56,10 @@ internal fun USBankAccountEmitters(
         usBankAccountFormArgs.handleScreenStateChanged(
             context = context,
             screenState = screenState,
-            enabled = hasRequiredFields,
+            enabled = hasRequiredFields && !screenState.isProcessing,
             merchantName = viewModel.formattedMerchantName(),
             onPrimaryButtonClick = viewModel::handlePrimaryButtonClick,
+            onPrimaryButtonStateChanged = viewModel::handlePrimaryButtonStateChanged
```

Comment:
Can we just call `handlePrimaryButtonStateChanged(StartProcessing)` from inside `handlePrimaryButtonClick`?

We also don’t seem to reset the processing state. Do we need to do that?

Reply:
We don't need to, it is handled when the bank account is returned from FC SDK, by setting the `isProcessing` flag to false.